test/createApolloMockedProvider.test.tsx

```diff
@@ -97,41 +88,30 @@ test('works with custom links', async () => {
   );
 });
 
-test('works with client resolvers', async () => {
-  const clientResolvers = {
-    Todo: {
-      text: () => 'client',
-    },
-  };
+// This test does not pass.
```

Comment:
Todo: do we keep this or get rid of clientResolvers?

Reply:
I say we remove it for now

Reply:
🗑️